test/remove_filename_quotes.tape.js

```diff
@@ -0,0 +1,27 @@
+var test = require('tape');
+var rewire = require('rewire');
```

Comment:
Probably should add these to `devDependencies` in package.json

Reply:
That's exactly why I didn't add test to scripts list in package. Test "suite" is just to small to run it at the moment, so I didn't want to make xdcc autoinstall two packages just for that. And when somebody wants to run them by hand, he can install those himself.

I left it to you as an author to decide if you want to install those or not.